assets/js/index.js

```diff
@@ -0,0 +1,97 @@
+let library = setupLibrary();
+
+if (!localStorage.uniqueId) {
+  localStorage.uniqueId = 1; 
+}
+
+function setupLibrary() {
```

Comment:
To avoid having to use the variable `library` here and thus make your function simpler:

```javascript
function setupLibrary() {
  if(localStorage.library) {
    return JSON.parse(localStorage.library).map((e) => Object.assign(new Book(), e));
  } 
  return [];
}
```